validator/sawtooth_validator/journal/publisher.py

```diff
@@ -125,10 +126,6 @@ def _build_block(self, chain_head):
         self._scheduler = self._transaction_executor.create_scheduler(
             self._squash_handler, chain_head.state_root_hash)
 
-        self._transaction_executor.execute(self._scheduler)
-        for batch in self._pending_batches:
-            self._validate_batch(batch)
-
         # build the TransactionCache
         self._committed_txn = TransactionCache(self._block_cache.block_store)
```

Comment:
This was probably my mis-reading the diff, and not seeing the fact that it was in a separate function.